### PR TITLE
Adding a file for Greece

### LINKED ISSUE #21: VasileiosOik Recommendations
## Things to do in Greece

Let's add a file for people who are visiting Greece for the first time!

### Steps for this project

- [x]  Create branch
- [x]  Create file
- [x]  Commit changes
- [x] Create pull request
- [ ] Review pull request

### PR DESCRIPTION
**Pull request by VasileiosOik**
This Pull Request resolves #21 by adding recommendations for places to eat and things to do in Greece.

To test this build locally ...

Please provide more information of the types of food you like and the things you enjoy doing when you travel.

### CODE REVIEW

**Comment by beardofedu** [comment]
@VasileiosOik I love going to Karpathos and having a 🍺 at the 🏖 . Any recommendations on a great place to get a good 🍺 .

Also the food is really good, pickled 🐙 is a favorite, any recommendations on where to get some?

File: githubvasileios-greece.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ## Greece
+
+ ### Great placs to eat
```
Comment (VasileiosOik) [inline githubvasileios-greece.md]: i have a different idea

File: githubvasileios-greece.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ ## Greece
```
Comment (VasileiosOik) [inline githubvasileios-greece.md]: any good ideas?

**Comment by paulfrankallan** [comment]
Some great ideas to start with!

**Pull request merged**